README.md

```diff
@@ -4,42 +4,29 @@ Seabird has been an IRC bot for the last 10 years in many different
 incarnations. This version is a gRPC service which exports a number of functions
 to easily interact with chat services.
 
-## Server Implementations
-
-There are currently two server implementations. They export the same interface
-and should respond almost exactly the same. They are both designed to use the
-same environment variables for configuration and the same config file format.
-Either one can be used interchangeably.
-
 This server acts only as an event broker - you will need both a running chat
 backend and some sort of plugin in order for anything visible to happen.
 
-### Go
-
-The Go version uses the official gRPC libraries for a server implementation but
-a semi-custom http Handler in order to support grpc-web.
-
-### Rust
-
-The Rust version uses `tonic` for a gRPC server implementation.
+The server is implemented in Rust and uses `tonic` for a gRPC server
+implementation.
 
 ## Building
 
 The easiest way to build and deploy `seabird-core` is to use the [official
-docker images](https://hub.docker.com/r/seabirdchat/seabird-core). The `go` and
-`rust` tags point to the Go and Rust implementations respectively on the master
-branch. Additionally, the `latest` tag will point to whichever implementation is
-given focus when implementing. This is currently the `rust` version.
+docker images](https://hub.docker.com/r/seabirdchat/seabird-core).
 
 In order to build these, you can use one of the following:
 
 ```sh
-docker build -t seabird-core:go -f Dockerfile-go .
-docker build -t seabird-core:rust -f Dockerfile-rust .
+docker build -t seabird-core:rust .
 ```
 
-If you want to run the server outside of docker, there is information in each
-implementation's sub-folder README.
+If you want to develop locally, simply use the normal cargo workflow in order to
+build/run seabird-core:
```

Comment:
Unrelated to this PR but might be worth adding a reminder to `git submodule update --init`